front/src/modules/ui/input/relation-picker/components/SingleEntitySelectBase.tsx

```diff
@@ -71,9 +76,22 @@ export function SingleEntitySelectBase<
   entitiesInDropdown = entitiesInDropdown.filter((entity) =>
     isNonEmptyString(entity.name.trim()),
   );
+  const theme = useTheme();
 
   return (
     <DropdownMenuItemsContainer ref={containerRef} hasMaxHeight>
+      {noUser && (
+        <DropdownMenuItem onClick={() => onEntitySelected(noUser)}>
+          {noUser.entityType === Entity.User ? (
```

Comment:
as we pass a noUser with label and icons, you don't have to do this if condition anymore :)